manager/gui/src/components/object-store-config/TenantSecurity.vue

```diff
@@ -0,0 +1,154 @@
+        </v-col>
+        <v-col cols="2">
+        </v-col>
+      </v-row>
+      <v-row class="field-row pl-4">
+        <v-col cols="3" class="field-label">
+          Enable AutoCert
+        </v-col>
+        <v-col cols="2" class="toggle-col">
+           <SgtToggle
+            :tooltip=" toggleStates.enableAutoCert ? 'On' : 'Off'"
+            v-model="toggleStates.enableAutoCert"
+            :labelPre = "txtOff"
+            :labelPost = "txtOn"
+            :areToggleLabel="true"
+          ></SgtToggle>
+        </v-col>
+      </v-row>
+      <v-row class="field-row pl-4">
+        <v-col cols="3" class="field-label">
+          Custom Certificates 
+        </v-col>
+        <v-col cols="2" class="toggle-col">
+           <SgtToggle
+            :tooltip=" toggleStates.enableCustomCert ? 'On' : 'Off'"
+            v-model="toggleStates.enableCustomCert"
+            :labelPre = "txtOff"
+            :labelPost = "txtOn"
+            :areToggleLabel="true"
+          ></SgtToggle>
+        </v-col>
+      </v-row>
+    </div>
+    
+    <div class="page-title">
+        Encryption <SgtTooltipIcon>How would you like to encrypt the information at rest.</SgtTooltipIcon> 
+    </div>
+    <v-divider></v-divider>
+    <div class="row-padding">
+      <v-row class="field-row">
+        <v-col cols="3" class="field-label">
+          Enable Server Side Encryption 
+        </v-col>
+        <v-col cols="2" class="toggle-col">
+           <SgtToggle
+            :tooltip=" toggleStates.enableServerSideEncryption ? 'On' : 'Off'"
+            v-model="toggleStates.enableServerSideEncryption"
+            :labelPre = "txtOff"
+            :labelPost = "txtOn"
+            :areToggleLabel="true"
+          ></SgtToggle>
+        </v-col>
+      </v-row>
+      </div>
+      <v-divider></v-divider>
+      <div class="row-padding">
+      <v-row class="field-row">
+        <v-col cols="4" class="button-col">
+          <v-btn
+            class="mr-5"
+            color="primary"
+            >Save
+          </v-btn>
+          <v-btn
+            color="csmdisabled"
+            depressed
+            dark
+            >Reset</v-btn
+          >
+        </v-col>
+      </v-row>
+      </div>
+    </div>
+</template>
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import SgtTooltipIcon from "@/lib/components/SgtTooltipIcon/SgtTooltipIcon.vue";
+import SgtToggle from "@/lib/components/SgtToggle/SgtToggle.vue";
+
+@Component({
+  name: "TenantSecurity",
+  components: { SgtTooltipIcon, SgtToggle},
+})
+export default class TenantSecurity extends Vue {
+  
+  toggleStates = {
+    enableTls: true,
+    enableAutoCert: true,
+    enableCustomCert: false,
+    enableServerSideEncryption: false,
+  };
+  txtOn = "ON";
+  txtOff = "OFF";
+
+  
+}
+</script>
+<style lang="scss" scoped>
+.row-padding{
+  padding: 20px 0 20px 20px;
+}
+.toggle-col{
+  font-weight: bold;
+}
+.pl-4{
+  .toggle-col{
+    margin-left: -11px;
```

Comment:
Try to avoid extra CSS and use predefined classes from Vuetify

Reply:
Used the classes from Vuetify.